packages/roosterjs-editor-plugins/lib/TabIndent/TabIndent.ts

```diff
@@ -113,4 +144,34 @@ export default class TabIndent implements EditorPlugin {
 
         return false;
     }
+
+    // Check if it is a blockquote event, if it is true, also return the blockquote element where the cursor resides
+    // 1. Cursor is in blockquote element
+    // 2. Current block has no content
+    // 3. is keyDown
+    // 4. is Enter
+    // 5. Any of ctrl/meta/alt is not pressed
+    private isBlockQuoteEvent(event: PluginEvent, interestedKeyCodes: number[], nodeAtCursor: Node): [boolean, Node] {
```

Comment:
Will interestedKeyCodes be changed in different scenarios? If not, why do you need to pass this parameter?

Reply:
Removed